admin/organization/index.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Learn about organizations.
+keywords: organizations, admin, overview
+title: Organization administration overview
+---
+
+{% include admin-early-access.md %}
+
```

Comment:
Is it worth adding the diagram here again? Like you did in the company section?

Reply:
Can do. I'll chop off the company and extra org.